joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/OperatorTests.scala

```diff
@@ -497,29 +497,6 @@ class OperatorTests extends PhpCode2CpgFixture {
     }
   }
 
-  "temporary list implementation should work" in {
```

Comment:
I think instead of completely removing this test, we should just change it to test what your AST looks like with the lowering